src/entity/EnemyShip.java

```diff
@@ -61,16 +63,16 @@ public class EnemyShip extends Entity {
 
 
 	public EnemyShip(final int positionX, final int positionY,
-			final SpriteType spriteType,int hp,int x, int y) {// Edited by Enemy
-		super(positionX, positionY, 12 * 2, 8 * 2, HpEnemyShip.determineColor(hp));
+					 final SpriteType spriteType, int hp, int x, int y) { // Edited by Enemy
+		super(positionX, positionY, getShipWidth(spriteType) * 2, getShipHeight(spriteType) * 2, HpEnemyShip.determineColor(hp));// return the hitbox to width and height
```

Comment:
I think this code should be edited, when we add real boss sprite on kakaotalk. But this logic is good too. GOOD!